### PR TITLE
add sl and env img

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr adds images to visually explain SL and dbt cloud env hierarchy concept. 

[diagram docs project](https://www.notion.so/dbtlabs/Phase-2-diagrams-to-confusing-or-complex-workflows-6696288a0c5742aeb4eb5819cb84fb20?pvs=4)

---
🚀 Deployment available! Here are the direct links to the updated files:

- https://docs-getdbt-com-git-add-env-img-dbt-labs.vercel.app/docs/dbt-cloud-environments
- https://docs-getdbt-com-git-add-env-img-dbt-labs.vercel.app/docs/use-dbt-semantic-layer/dbt-sl
- https://docs-getdbt-com-git-add-env-img-dbt-labs.vercel.app/docs/use-dbt-semantic-layer/sl-faqs

### CODE REVIEW

File: website/docs/docs/use-dbt-semantic-layer/dbt-sl.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ The dbt Semantic Layer, powered by [MetricFlow](/docs/build/about-metricflow), s
 
 Moving metric definitions out of the BI layer and into the modeling layer allows data teams to feel confident that different business units are working from the same metric definitions, regardless of their tool of choice. If a metric definition changes in dbt, it’s refreshed everywhere it’s invoked and creates consistency across all applications. To ensure secure access control, the dbt Semantic Layer implements robust [access permissions](/docs/use-dbt-semantic-layer/setup-sl#set-up-dbt-semantic-layer) mechanisms.
 
+<Lightbox src="/img/docs/dbt-cloud/semantic-layer/sl-concept.png" width="90%" title="This diagram shows how the dbt Semantic Layer works with your data stack." />
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/dbt-sl.md]:
```suggestion
<Lightbox src="/img/docs/dbt-cloud/semantic-layer/sl-concept.png" width="80%" title="This diagram shows how the dbt Semantic Layer works with your data stack." />
```

File: website/docs/docs/dbt-cloud-environments.md [hunk 2]
```diff
@@ -15,8 +15,9 @@ Critically, in order to execute dbt, environments define three variables:
 
 Each dbt Cloud project can have only one [development environment](#create-a-development-environment), but there is no limit to the number of [deployment environments](/docs/deploy/deploy-environments), providing you the flexibility and customization to tailor the execution of scheduled jobs. 
 
-Use environments to customize settings for different stages of your project and streamline the execution process by using software engineering principles. This page will detail the different types of environments and how to intuitively configure your development environment in dbt Cloud. 
+<Lightbox src="/img/dbt-env.png" width="90%" title="dbt Cloud environment hierarchy showing projects, environments, connections, and orchestration jobs." />
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-environments.md:20]:
```suggestion
Use environments to customize settings for different stages of your project and streamline the execution process by using software engineering principles. 

<Lightbox src="/img/dbt-env.png" width="90%" title="dbt Cloud environment hierarchy showing projects, environments, connections, and orchestration jobs." />
```

File: website/docs/docs/dbt-cloud-environments.md [hunk 3]
```diff
@@ -15,8 +15,9 @@ Critically, in order to execute dbt, environments define three variables:
...
 
+Use environments to customize settings for different stages of your project and streamline the execution process by using software engineering principles. This page will detail the different types of environments and how to intuitively configure your development environment in dbt Cloud.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-environments.md]: I wonder if this 1st sentence can move above the image to help describe what the image is conveying.
I mistook "this page" to mean the image at first. Maybe rewording might help.

```suggestion
The following sections detail the different types of environments and how to intuitively configure your development environment in dbt Cloud. 
```

**Review by runleonarun — APPROVED** [review]
Approving with two minor / non-blocking comments that have little to do with the diagrams 😆
